js/core/config.js

```diff
@@ -22,6 +22,12 @@ var config = {
     * @type string
     */
     defaultCurrency: "USD",
+    /**
+    * @name globalConfig.stringToLower
```

Comment:
The property name does not tell that is it related to.

Reply:
The "stringToLower" name correlates with [stringToLower](https://devexpress.github.io/DevExtreme.AspNet.Data/net/api/DevExtreme.AspNet.Data.DataSourceLoadOptionsBase.html#DevExtreme_AspNet_Data_DataSourceLoadOptionsBase_StringToLower)  in ASPNET.Data. 

That's why, we decided to name it in the same way. For global config, was decided to add "oData" prefix.

Reply:
Considering the "Remove stringToLower from load optioins" commit, I suggest another option name - "filterToLower". I think it is a bit clearer than "stringToLower".
@nightskylark  @AlekseyMartynov @dxvladislavvolkov  What do you think?

Reply:
I suggest one of these alternate names:
```suggestion
    * @name globalConfig.lowerizeFilterValue
```
or even:
```suggestion
    * @name globalConfig.useCaseSensitiveFilter
```

Reply:
Or maybe:
```suggestion
    * @name globalConfig.filterCaseSensitivity
```
Like `Qt` [does](http://doc.qt.io/qt-5/qsortfilterproxymodel.html#filterCaseSensitivity-prop).

Reply:
Seems that "filterToLower"/"oDataFilterToLower" should be quite clear, because OData has the "tolower" function
5.1.2.4.8. tolower see [here](https://www.odata.org/documentation/odata-version-3-0/url-conventions/)